watertap/unit_models/reverse_osmosis_base.py

```diff
@@ -370,6 +370,64 @@ def eq_flux_mass(b, t, x, p, j):
                     - prop_perm.conc_mass_phase_comp[p, j]
                 )
 
+        if self.config.transport_model == "SKK":
+
+            self.eq_flux_mass.deactivate()
+
+            self.reflect_coeff = Var(
```

Comment:
I figured this could be added in whether it's SKK or sol-diff. For SKK, leave as unfixed var. For sol-diff, fix to 1.

Reply:
Addressed in commit d914a77